contents/docs/surveys/webhook.mdx

```diff
@@ -38,3 +40,49 @@ Save your action and you will now begin receiving notifications whenever users r
     alt="How to create an action for survey responses" 
     classes="rounded"
 />
+
+## Using realtime destinations
+
+Our new [realtime destinations](/docs/cdp/destinations) are currently in preview. These provide customizable destinations, more formatting options, a revamped configuration UI. We can use them to send survey responses to Slack or any other destination that supports webhooks.
```

Comment:
```suggestion
Our new [realtime destinations](/docs/cdp/destinations) are currently in preview. These provide customizable destinations, more formatting options, a revamped configuration UI. You can use them to send [survey](/docs/surveys) responses to Slack or any other destination that supports webhooks.
```

Reply:
Is this only for survey responses?

Reply:
Since this is in the surveys section, I just talk about surveys here.